### ISSUE TITLE
First connection issues

### ISSUE THREAD

**Issue by boucadair** [danwing/authenticating-network#19]
A rogue AP may advertise a valid encrypted DNS server. The host may successful connect to that AP and stores the information. When the hosts connect to a legitimate network with the same credentials, the check will fail and access to the legitimate AP fails.

**Comment by danwing** [comment]
Yes, this is a problem of any TOFU scheme such as described in the I-D.

The only mitigation so far is to tie the SSID's name to the DNS server name (e.g., SSID="example.com <http://example.com/>" to require the DNS name to be "*.example.com <http://example.com/>"), which is mentioned in the I-D.
